apple/internal/testing/apple_test_rules.bzl

```diff
@@ -102,6 +102,9 @@ requirements for this test.
 """,
         "test_environment": """
```

Comment:
same here, should we just remove this field since it's not being used anymore?

Reply:
Originally I think you recommended leaving this for backwards compatibility but I may have misunderstood. I'll remove this

Reply:
now that you mention, that rings a bell, but still, I'll migrate internal runners and post this in release notes so that other external clients know to update their runners, I realized I don't want to track a migration path for this